cljfmt/src/cljfmt/core.cljc

```diff
@@ -341,6 +341,25 @@
   ([form indents alias-map]
    (transform form edit-all should-indent? #(indent-line % indents alias-map))))
 
+(defn split-maps
+  "Puts each hashmap pair on it's own line"
+  [form]
+  (transform form edit-all z/map?
```

Comment:
Are we searching for `z/map?`, or would a custom function like `map-value?` be a better target? If you write a search function that finds map values, then the logic for transforming becomes simpler in turn.